views/secure-register-filter.html

```diff
@@ -24,7 +26,7 @@
           name: "is_secure_register",
           fieldset: {
             legend: {
-              text: "Do any beneficial owners or managing officers have their personal information protected at Companies House?",
```

Comment:
Do you need to change the secure register controller in the registration journey to also pass in the title?

Reply:
The register controller wont need to be changed. The title is being set on the HTML document itself so the register journey controller requires no changes.